<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiSelectedFilesWidgetAdd</class>
 <widget class="QWidget" name="GuiSelectedFilesWidgetAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>86</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>Selected Files</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTreeView" name="_SelectedFilesTreeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>176</width>
       <height>110</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>87</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>87</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="_BtAdd">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_BtAddAll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Add All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="_BtRemove">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
